packages/ui/src/machines/authenticator/defaultServices.ts

```diff
@@ -49,6 +54,10 @@ export const defaultServices = {
   }): Promise<any> {
     return await Auth.confirmSignUp(username, code);
   },
+  /** @deprecated For internal use only */
+  async handleResendConfirmationCode({ username }: { username: string }) {
```

Comment:
I do not think the argument gets passed in is an object. You should not destructure here

Reply:
Oh yeah, the arguments should match the underlying auth call. So +1 with

```suggestion
  async handleResendConfirmationCode(username: string) {
```